src/processor/operator/persistent/insert_executor.cpp

```diff
@@ -157,6 +157,13 @@ void RelInsertExecutor::insert(transaction::Transaction* tx, ExecutionContext* c
     writeResult();
 }
 
+void RelInsertExecutor::skipInsert(kuzu::processor::ExecutionContext* context) {
+    for (auto i = 1u; i < columnDataEvaluators.size(); ++i) {
```

Comment:
Can we refactor this for loop to a function called evaluate result?

Reply:
I will do a separate PR for this. I think there is more to change if we were to provide a base `InsertExecutor`